doc/ref_cert/RC1/chapters/chapter03.md

```diff
@@ -88,7 +88,7 @@ They cover all together the API testing requirements as asked by
 [RA1 Core OpenStack Services APIs](../../../ref_arch/openstack/chapters/chapter05.md)
 
 According to Wallaby, the following software versions are considered here to
```

Comment:
@collivier "According to Wallaby" seems redundant with "verify OpenStack Wallaby"

Reply:
Agree. It was just an update of the previous redundancy. updating

Reply:
done thx